native/src/components/FeedbackContainer.tsx

```diff
@@ -134,7 +137,10 @@ const FeedbackContainer = (props: PropsType): ReactElement => {
       }
     })
     request().catch(err => {
-      console.log(err)
+      console.error(err)
+      if (fromError(err) !== ErrorCode.NetworkConnectionFailed) {
+        Sentry.captureException(err)
```

Comment:
```suggestion
        reportError(err)
```

Reply:
This was in native, in native I did not create a reportError function